src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -220,6 +221,32 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        protected override bool OnShutdown(int timeoutMilliseconds)
+        {
+            bool? result;
+            var sw = Stopwatch.StartNew();
+            this.listener?.Dispose();
```

Comment:
It is a bit strange to call dispose in shutdown.
Consider clean up the dispose and shutdown methods (e.g. Dispose will dispose the underlying listener).

Reply:
The spec says that after the TracerProvider Shutdown, any subsequent calls to get a Tracer should result in a no-op tracer. To achieve the equivalent of that requirement, we have to dispose the listener. If we don't dispose the listener we would continue to export the data as the exporters don't implement OnShutdown.